asf_search/search/search.py

```diff
@@ -127,6 +126,9 @@ def search(
     for key in listify_fields:
         if key in data and not isinstance(data[key], list):
             data[key] = [data[key]]
+    
+    if 'intersectsWith' in list(data.keys()):
```

Comment:
Somewhere in here, how we're saving the repaired wkt changed. It repairs a 4-D wkt, and saves the 3D version:
```python
import asf_search
opts = asf_search.ASFSearchOptions(platform="S1", intersectsWith="POINT(0 0 0 0)", maxResults=1)
results = asf_search.search(opts=opts)
results.searchOptions.intersectsWith
# Shows: 'POINT Z (0.0000000000000000 0.0000000000000000 0.0000000000000000)'
```

Reply:
Now properly saves as 2D